snap/snapcraft.yaml

```diff
@@ -1,25 +1,39 @@
-name: srsran
+name: srsran-4g
 base: core20
```

Comment:
Do you think it would make sense to update the base to something more up to date?